src/controller/OnOffCluster.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app/chip-zcl-zpro-codec-api.h>
+#include <controller/OnOffCluster.h>
+
+namespace chip {
+namespace Controller {
+
+constexpr uint16_t kMaxOnOffMessageLength = 64;
```

Comment:
Maybe you want to reference https://github.com/project-chip/connectedhomeip/issues/965 here